_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -64,14 +65,26 @@ sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86
 {% endhighlight %}
 </div>
 
-<div class="quickstart_os quickstart_os_el7">
+<div class="quickstart_os quickstart_os_centos7">
+  <p>Enable the EPEL (Extra Packages for Enterprise Linux), Foreman and SCL (Software Collections) repositories:</p>
+
+{% highlight bash %}
+sudo yum-config-manager --enable extras
+sudo yum -y install epel-release centos-release-scl-rh
+sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
+{% endhighlight %}
+
+</div>
+
+<div class="quickstart_os quickstart_os_other_el7">
   <p>Enable the EPEL (Extra Packages for Enterprise Linux), Foreman and SCL (Software Collections) repositories:</p>
 
 {% highlight bash %}
 sudo yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
-sudo yum -y install foreman-release-scl
+sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/centos-release-scl-rh-2-3.el7.centos.noarch.rpm
```

Comment:
Please use https. Also the epel link that was there before can use https (like we do in the RHEL instructions).